hackDetail.js

```diff
@@ -13,15 +16,39 @@ export const detail = async (event, context) => {
     });
     conn.model(
       "Hack",
-      new mongoose.Schema({ title: String, description: String, goal: String, creator: String, team: Array })
+      new mongoose.Schema({
+        title: String,
+        description: String,
+        goal: String,
+        team: Array,
+        likes: [mongoose.ObjectId]
+      })
     );
     conn.model("User", new mongoose.Schema({ name: String, email: String }));
   }
   const Hack = conn.model("Hack");
+  let hasUserLiked = false;
+  let numberLikes = 0;
+
+  if (userId !== "null") {
+    //that means user is logged in and we have his id
+
+    let result = await Hack.findById(id);
+    numberLikes = result.likes.length;
```

Comment:
We are deriving this (https://github.com/hackthehack/hack-zero-v2-backend/pull/26/files#r403772993)
what are we using stored value for?

Reply:
to send back to the frontend about the how many likes for that particular hack.

Reply:
thats this derived value thats going to the front end (line 58), I was talking about the value that were storing in the database, but you've resolved that now.